app/controllers/admin/services_controller.rb

```diff
@@ -2,13 +2,24 @@ module Admin
 
   class ServicesController < ApplicationController
 
-    skip_before_action :verify_authenticity_token, only: :cancel_reservations_for_offline_instruments
+    skip_before_action :verify_authenticity_token
 
     def cancel_reservations_for_offline_instruments
       InstrumentOfflineReservationCanceler.new.cancel!
       render nothing: true
     end
 
+    def process_five_minute_tasks
+      self.class.five_minute_tasks.each do |worker|
+        worker.new.perform
+      end
+      render nothing: true
+    end
+
+    def self.five_minute_tasks
+      @five_minute_tasks ||= [AutoExpireReservation, EndReservationOnly, AutoLogout]
```

Comment:
Options: 

1) We change Autologout to match and have it trigger the background notification.
2) Don't catch anything in the individual jobs and instead catch at the controller level.

I need to think a little bit, but I think I lean towards 2. That isolates the error handling into one place. The rollbacks will still happen because an exception is still getting raised; it'll get re-raised by the transaction after rolling back.